### ISSUE TITLE
Login screen doesn't load

### ISSUE THREAD

**Issue by cyrilschreiber3** [Yooooomi/your_spotify#154]
Hi, I'm trying to set up YourSpotify on docker with partial success. 
When I go to `https://my.domain/login` I get the dashboard with the "Login with Spotify" button but when I click on it I just get a blank dashboard and get the url: `https://my.domain/api/oauth/spotify` and neither my browser nor Apache gives me errors.

My redirect url is `https://my.domain/api/oauth/spotify/callback`
I'm using Docker on Windows, IDK if it changes anything

Here is my docker-compose.yml
```yml
version: "3"

services:
  server:
    container_name: your_spotify_server
    image: yooooomi/your_spotify_server
    restart: unless-stopped
    ports:
      - "5100:8080"
    links:
      - mongo
    depends_on:
      - mongo
    environment:
      - API_ENDPOINT=https://my.domain/api
      - CLIENT_ENDPOINT=https://my.domain
      - SPOTIFY_PUBLIC=
      - SPOTIFY_SECRET=
      - CORS=https://my.domain
      - MONGO_ENDPOINT=mongodb://mongo:27017/your_spotify
  
  mongo:
    container_name: mongo_spotify
    image: mongo:4
    volumes:
      - ./database:/data/db

  web:
    container_name: your_spotify_client
    image: yooooomi/your_spotify_client
    restart: unless-stopped
    ports:
      - "5000:3000"
    environment:
      - API_ENDPOINT=https://my.domain/api
```

And here is my apache2 reverse proxy config:

```ApacheConf
<IfModule mod_ssl.c>
<VirtualHost *:443>
	ServerName my.domain
	ErrorLog /var/www/my.domain/log/error.log
	CustomLog /var/www/my.domain/log/requests.log combined

	RewriteEngine On
	SSLProxyEngine On
	SSLProxyCheckPeerCN Off
	SSLProxyCheckPeerName Off
	SSLProxyVerify none

	ProxyPass "/" http://192.168.1.113:5000/
	ProxyPassReverse "/" http://192.168.1.113:5000/

	ProxyPass "/api" http://192.168.1.113:5100/
	ProxyPassReverse "/api" http://192.168.1.113:5100/
	ProxyPreserveHost On

SSLCertificateFile /etc/letsencrypt/live/my.domain/fullchain.pem
SSLCertificateKeyFile /etc/letsencrypt/live/my.domain/privkey.pem
Include /etc/letsencrypt/options-ssl-apache.conf
</VirtualHost>
</IfModule>
```

I know there are a lot of issues about that, but none of them resolved my problem.
Any ideas ?

**Comment by Yooooomi** [comment]
Hello! Are you sure your path gets rewrote once reaching the backend? If it is not then the server returns a 404 and does not redirect you to the application.
TLDR: The /api should become / when reaching the server.

**Comment by cyrilschreiber3** [comment]
Hello ! Thank you very much, it was actually that. But since I don't know how to rewrite the path and I don't to put the effort into doing that, I just used two domain names. Like `my.domain` and `api.my.domain`.
